src/modules/BootGRUB2.rb

```diff
@@ -152,6 +152,9 @@ def Propose
         partition = prep_partitions.first
         if partition
           BootCommon.globals["boot_custom"] = partition
+          BootCommon.globals["boot_boot"] = "false"
+          BootCommon.globals["boot_root"] = "false"
+          BootCommon.globals["boot_mbr"] = "false"
```

Comment:
it is also handled in stage1 proposal...so for ppc I suggest to use custom prep partition, activate flag and do not set generic_mbr there. It at least unify behavior and similar stuff for s390. Reason why it is separated now, is that stage1 originally in SLE12 GA is used only on x86 architecture, so I refactoring it also this way, but it is more correct to do it properly.